hazelcast/src/main/java/com/hazelcast/jet/config/JobConfig.java

```diff
@@ -350,7 +350,7 @@ public JobConfig addPackage(@Nonnull String... packages) {
      * <p>
      * This variant identifies the JAR with a URL, which must contain at least one
      * path segment. The last path segment ("filename") will be used as the resource
-     * ID, so two JARs with the same filename will be in conflict.
+     * ID - and will not be replaced if already exists.
```

Comment:
I think we can be even more clear about this, something along the lines of:
```suggestion
     * ID and replacing an existing resource with a new version using the same ID is
     * <b>not supported</b> at runtime.
```
Wdyt?